ansible/roles/anitya-dev/files/check-service.service

```diff
@@ -5,7 +5,7 @@ Documentation=https://github.com/fedora-infra/anitya/
 
 [Service]
 Environment="ANITYA_WEB_CONFIG=/home/vagrant/anitya.toml"
-ExecStart=check_service.py
+ExecStart=/home/vagrant/devel/anitya/check_service.py
```

Comment:
You shouldn't need that the script should be in standard PATH, it is installed as binary

Reply:
yes, you are right, changed path to the script, provided by binary install